build.assets/Dockerfile

```diff
@@ -293,7 +293,8 @@ RUN helm plugin install https://github.com/vbehar/helm3-unittest && \
 # Install JS gRPC tools.
 ARG NODE_GRPC_TOOLS_VERSION # eg, "1.12.4"
 ARG NODE_PROTOC_TS_VERSION  # eg, "5.0.1"
-RUN npm install --global "grpc-tools@$NODE_GRPC_TOOLS_VERSION" "grpc_tools_node_protoc_ts@$NODE_PROTOC_TS_VERSION"
+ARG PROTOBUF_TS_PLUGIN_VERSION  # eg, "2.9.3"
+RUN npm install --global "grpc-tools@$NODE_GRPC_TOOLS_VERSION" "grpc_tools_node_protoc_ts@$NODE_PROTOC_TS_VERSION" "@protobuf-ts/plugin@$PROTOBUF_TS_PLUGIN_VERSION"
```

Comment:
Is this needed in the Ci container? We're not using it to generate proto files.

Reply:
Yes, the usage is in a follow up PR